docs/applications.md

```diff
@@ -85,6 +85,64 @@ sharing the same configuration across observers, fluo provides a simple
 mechanism to set and access application specific configuration.  See the
 javadoc on [FluoClient].getAppConfiguration() for more details.
 
+Debugging Applications
+======================
+
+When monitoring [metrics][metrics] and a problem with an application is
+detected, like too many transaction collisions, it can be difficult to figure
+out what is causing the problem.   Low level logging about transactions may
+help track the issue down by showing the specific data involved.  Configuring
+the following Loggers to TRACE will log detailed information about
+transactions.
+
+| Logger                  | Level | Information                                                                                        |
+|-------------------------|-------|----------------------------------------------------------------------------------------------------|
+| `io.fluo.tx`            | TRACE | Provides detailed information about what transactions read and wrote                               |
```

Comment:
Does this setting this to TRACE also set `tx.summary` & `tx.collision` to TRACE?